docs/extensions-plugins/cli/commands.mdx

```diff
@@ -72,13 +81,21 @@ Lists all accessible AI models.
 pieces list models
 ```
 
-### `open`
-Opens an asset from a list or search. If only "open" is used, it opens the most recent asset.
+### `open` (deprecated)
+Opens an asset from a list or search. If only "open" is used, it opens the most recent asset. This also creates a link to the asset's code.
 
 ```bash
 pieces open [ITEM_INDEX]
 ```
 
+ ##### Editor Configuration:
```

Comment:
If you could also update this to be an h4 that would be great!

Reply:
yup , that should have always been h4 , apologies for the typo , fixed in commit [9e734b5](https://github.com/pieces-app/documentation/pull/538/commits/9e734b5069f55b71f40ed9a14ff6c0a1b0b171b2)